docs/common/top-bar.tsx

```diff
@@ -7,16 +7,17 @@ interface Props {
 }
 
 const TopBar: React.FC<Props> = ({ icon, title }) => {
+  const { palette } = useTheme();
   return (
     <Box
       sx={{
         height: '68px',
-        bgcolor: '#555555',
+        bgcolor: `${palette.grey['700']}`,

```

Comment:
Exactly what I was looking for with the use of the pallet, but do you need to wrap it in `${}`?  Should should be able to just use `bgcolor: palette.grey['700']`

Reply:
I've removed the `${}` and used directly the value from palette.